src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticateResult.cs

```diff
@@ -31,12 +31,17 @@ public class AuthenticateResult
         /// <summary>
         /// Additional state values for the authentication session.
         /// </summary>
-        public AuthenticationProperties Properties => Ticket?.Properties;
+        public AuthenticationProperties Properties => Ticket?.Properties ?? Error?.Properties;
```

Comment:
Ugh, get rid of this fallback, I think Properties should only be Ticket.Properties, make them go Error.Properties if they want the error ones.